packages/contracts/src/core/dao/DAO.sol

```diff
@@ -67,16 +67,19 @@ contract DAO is
     uint256 private constant _ENTERED = 2;
 
     /// @notice The [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) signature validator contract.
+    /// @dev Added in v1.0.0.
     IERC1271 public signatureValidator;
 
     /// @notice The address of the trusted forwarder verifying meta transactions.
+    /// @dev Added in v1.0.0.
```

Comment:
Adding this is nice, but if there are too many comments, code readability can become an issue in the future. Fine for now but let's think of other tools can help determining when something was added

Reply:
The good thing is, that this will show up in our dev portal